libsplinter/src/admin/store/diesel/models.rs

```diff
@@ -24,7 +24,7 @@ use diesel::{
     deserialize::{self, FromSql},
     expression::{helper_types::AsExprOf, AsExpression},
     serialize::{self, Output, ToSql},
-    sql_types::SmallInt,
+    sql_types::Integer,
```

Comment:
For which backend is this the case?

Reply:
Postgres was producing errors where the `SmallInt` type was unable to be converted, and would always return a `0` despite the column being set to an actual value (ie, 1). This resulted in an error from the `FromSql` implementation that would make it unable to view proposals at all.

Reply:
Changing the postgres migration file from `INTEGER` to `SMALLINT` has solved the error described in this commit, so I added that change to the initial commit that also sets the default in the migration files. And removed the commit that changed the representation from an i16 -> i32